examples/resources/adfs-connector.yaml

```diff
@@ -5,6 +5,14 @@ version: v2
 metadata:
   name: "adfs_connector"
 spec:
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "login with SSO"
```

Comment:
Capital L, `Login with SSO`? Or maybe `Login with ADFS`, but I don't know Windows applications refer to it as SSO or ADFS?